hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/instance/HazelcastClientLoader.java

```diff
@@ -77,6 +78,7 @@ public void configure(final Properties props) {
         clientConfig.getConnectionStrategyConfig().getConnectionRetryConfig().setInitialBackoffMillis(INITIAL_BACKOFF_MS);
```

Comment:
If we make timeout configurable, we may also consider making these values configurable as well. Because the current configuration overrides backoff and multiplier values with defaults regardless of whether they are configured via hazelcast-client.xml or not.

Reply:
I agree :) will cover in a separate PR